contracts/utils/FuseFixer.sol

```diff
@@ -0,0 +1,189 @@
+        address(0xe14c2e156A3f310d41240Ce8760eB3cb8a0dDBE3), // Pool 156: USTw
+        address(0x001E407f497e024B9fb1CB93ef841F43D645CA4F), // Pool 156: FEI
+        address(0x5CaDc2a04921213DE60B237688776e0F1A7155E6), // Pool 156: FRAX
+        address(0x9CD060A4855290bf0c5aeD266aBe119FF3b01966), // Pool 156: DAI
+        address(0x74897C0061ADeec84D292e8900c7BDD00b3388e4), // Pool 156: LUSD
+        address(0x88d3557eB6280CC084cA36e425d6BC52d0A04429), // Pool 156: USDC
+        address(0xe92a3db67e4b6AC86114149F522644b34264f858) // Pool 156: ETH
+    ];
+
+    mapping(address => address[]) internal underlyingsToCTokens;
+
+    constructor(address core) CoreRef(core) {
+        _buildCTokenMapping();
+
+        // check mappings lengths - hand calculated
+        assert(underlyingsToCTokens[UNDERLYINGS[0]].length == 7);
+        assert(underlyingsToCTokens[UNDERLYINGS[1]].length == 5);
+        assert(underlyingsToCTokens[UNDERLYINGS[2]].length == 5);
+        assert(underlyingsToCTokens[UNDERLYINGS[3]].length == 1);
+        assert(underlyingsToCTokens[UNDERLYINGS[4]].length == 5);
+        assert(underlyingsToCTokens[UNDERLYINGS[5]].length == 3);
+        assert(underlyingsToCTokens[UNDERLYINGS[6]].length == 2);
+        assert(underlyingsToCTokens[UNDERLYINGS[7]].length == 2);
+        assert(underlyingsToCTokens[UNDERLYINGS[8]].length == 4);
+        assert(underlyingsToCTokens[UNDERLYINGS[9]].length == 1);
+
+        // send out token approvals
+        _approveCTokens();
+    }
+
+    /// @dev Repay all debt
+    function repayAll() public onlyTribeRole(TribeRoles.PCV_GUARDIAN_ADMIN) {
+        _repayETH();
+
+        // we skip index 0 because that's ETH
+        for (uint256 i = 1; i < UNDERLYINGS.length; i++) {
+            _repayERC20(UNDERLYINGS[i]);
+        }
+    }
+
+    /// @dev Repay the underlying asset on all ctokens up to the maximum provide
+    /// @notice reverts if the total bad debt is beyond the provided maximum
+    /// @param underlying the asset to repay in
+    /// @param maximum the maximum amount of underlying asset to repay
+    function repay(address underlying, uint256 maximum) public onlyTribeRole(TribeRoles.PCV_GUARDIAN_ADMIN) {
+        require(getTotalDebt(underlying) < maximum, "Total debt is greater than maximum");
+
+        if (underlying == address(0)) {
+            _repayETH();
+        } else {
+            _repayERC20(underlying);
+        }
+    }
+
+    /* Helper Functions */
+
+    // Creates mappings of underlyings to all applicable ctokens
+    function _buildCTokenMapping() internal {
+        for (uint256 i = 0; i < CTOKENS.length; i++) {
+            address token = CTOKENS[i];
+            address underlying = CTokenFuse(token).underlying();
+            underlyingsToCTokens[underlying].push(token);
+        }
+    }
+
+    function _approveCTokens() internal {
+        // UNDERLYINGS[0] == ETH, so we skip that one
+        for (uint256 i = 1; i < UNDERLYINGS.length; i++) {
```

Comment:
I get why you do this, but feel it's leaving a footgun - if you change the `UNDERLYINGS` order around you'll skip the wrong `cToken`. 

Would personally check for `underlying != address(0x0)` or similar to be more explicit. One time use, so no strong feelings

Reply:
Done.